plugins/oauth/components/LoginPage.js

```diff
@@ -48,6 +48,9 @@ class LoginPage extends mixin(StoreMixin) {
 
   onMessageReceived(event) {
     if (event.origin !== SDK.config.authHost) {
+      console.warn(`Event Origin "${event.origin}" does not match
+        allowed origin "${SDK.config.authHost}"`);
```

Comment:
The extra whitespace in this string will be visible in the console, so let's just put it all on a single line.